src/wp-includes/class-wp-roles.php

```diff
@@ -138,20 +138,20 @@ public function reinit() {
 	}
 
 	/**
-	 * Add role name with capabilities to list.
+	 * Adds a role name with capabilities to the list.
```

Comment:
Should the docs changes for `add_role()` be included in this PR? Or will they be cherrypicked and committed separately?

Reply:
Good call, I've committed these changes separately in https://core.trac.wordpress.org/changeset/53436.